README.md

```diff
@@ -31,6 +31,7 @@ We release a new version of WooCommerce Blocks onto WordPress.org every few week
 1. Make sure you have WordPress 5.0+ and WooCommerce 3.6+
 2. Get a copy of this plugin using the green "Clone or download" button on the right.
 3. `npm install` to install the dependencies.
+4. `composer install` to install core dependencies.
 4. `npm run build` (build once) or `npm start` (keep watching for changes) to compile the code.
```

Comment:
Can you update the enumeration here? Currently there are two steps 4. :slightly_smiling_face: